indico/modules/groups/core.py

```diff
@@ -174,7 +178,9 @@ def as_legacy_group(self):
         return LocalGroupWrapper(self.id)
 
     def has_member(self, user):
-        return user and self.group in user.local_groups
+        if config.LOCAL_GROUPS:
```

Comment:
like above, I'd return false if local groups are disabled and then continue with the normal logic afterwards